src/Jackalope/RepositoryFactoryJackrabbit.php

```diff
@@ -93,6 +93,9 @@ public function getRepository(array $parameters = null)
         if (isset($parameters['jackalope.check_login_on_server'])) {
             $transport->setCheckLoginOnServer($parameters['jackalope.check_login_on_server']);
         }
+        if (isset($parameters['jackalope.force_http_version_10'])) {
+            $transport->forceHttpVersion10($parameters['jackalope.force_http_version_10']);
+        }
```

Comment:
afaik you also need to allow this parameter https://github.com/jackalope/jackalope-jackrabbit/blob/force_http_10/src/Jackalope/RepositoryFactoryJackrabbit.php#L37